iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/checks/CapabilitiesCheckTest.java

```diff
@@ -44,6 +44,13 @@ void shouldVerifyHelmArrayValues() {
     KubernetesVerifier.verify("CapabilitiesCheck/CapabilitiesChart/templates/capabilities-pod.yaml", check);
   }
 
+  @Test
+  void shouldNotHighlightIncorrectLine() {
+    // see https://sonarsource.atlassian.net/browse/SONARIAC-1308
+    var expectedIssue = issue(13, 0, 13, 45, "Make sure setting capabilities is safe here.");
```

Comment:
Checked that reverting https://sonarsource.atlassian.net/browse/SONARIAC-1357 shifts the location to the incorrect line. Incorrect highlighting of both lines must have been fixed even earlier.